contracts/src/v0.8/dev/KeeperRegistryLogic.sol

```diff
@@ -135,15 +107,102 @@ contract KeeperRegistryLogic is KeeperRegistryBase {
   /**
    * @dev Called through KeeperRegistry main contract
    */
+  function setConfig(
+    address[] memory signers,
+    address[] memory transmitters,
+    uint8 f,
+    bytes memory onchainConfig,
+    uint64 offchainConfigVersion,
+    bytes memory offchainConfig
+  ) external override onlyOwner {
+    if (signers.length > maxNumOracles) revert TooManyOracles();
+    if (f == 0) revert IncorrectNumberOfFaultyOracles();
+    if (signers.length != transmitters.length || signers.length <= 3 * f) revert IncorrectNumberOfSigners();
+    if (onchainConfig.length != 0) revert OnchainConfigNonEmpty();
```

Comment:
maybe it isn't necessary to actually store data in this field, but I thought we could use it for the RegistryConfig

Reply:
downstream we populate it with the encoding of actual params. don't quite like this but copied this pattern from other OCR contracts, need to think more